system-test/busybench-js/src/busybench.js

```diff
@@ -15,7 +15,9 @@
  */
 
 const fs = require('fs');
+// eslint-disable-next-line node/no-extraneous-require
 const pify = require('pify');
+// eslint-disable-next-line node/no-missing-require
```

Comment:
I'm a bit puzzled - is 'pprof' missing at this point? If not, why does gts v3 think so?

Reply:
I think gts may think pprof is missing because it's not mentioned in the package.json (this was something I had to add to make the linter pass).

Reply:
My best guess is that pprof is not in the package.json (I was also confused here, but added it to make the linter pass).